apps/smart-contracts/core/test/unit/PrePOMarket.test.ts

```diff
@@ -739,6 +739,26 @@ describe('=> prePOMarket', () => {
       expect(await collateralToken.balanceOf(user.address)).to.eq(totalOwed)
     })
 
+    it('does not burn short tokens if only long tokens are redeemed', async () => {
```

Comment:
nit:
```suggestion
    it('doesn't burn short tokens if only long tokens redeemed', async () => {
```
same for below